android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -26,8 +26,6 @@
 import android.text.Html;
 import android.text.InputType;
```

Comment:
Not related to this PR, but would you be able to remove the following imports as we are on this file:
`import android.text.InputType;`
`import android.os.Build;`
They don't seem to be used in the file anymore.

Reply:
Thanks for catching that. I'll remove them.